src/prefect/tasks/spacy/spacy_tasks.py

```diff
@@ -0,0 +1,216 @@
+from prefect import Task
+from prefect.utilities.tasks import defaults_from_attrs
+
+import spacy
+
+
+class SpacyNLP(Task):
+    """
+    Task for processing text with a spaCy pipeline.
+
+    Args:
+        - text (unicode, optional): string to be processed, can be provided during construction
+            or when task is run
+        - nlp (spaCy text processing pipeline, optional): a custom spaCy text
+            processing pipeline, if provided, this pipeline will be used instead
+            of being created from spacy_model_name
+        - spacy_model_name (str, optional): name of the spaCy language model, default
+            model is 'en_core_web_sm', will be ignored if nlp is provided
+        - disable (List[str], optional): list of pipeline components
+            to disable, only applicable to pipelines loaded from spacy_model_name
+        - component_cfg (dict, optional): a dictionary with extra keyword
+            arguments for specific components, only applicable to pipelines loaded from
+            spacy_model_name
+        - **kwargs (dict, optional): additional keyword arguments to pass to the
+            Task constructor
+    """
+
+    def __init__(
+        self,
+        text: str = u"",
+        nlp=None,
+        spacy_model_name: str = "en_core_web_sm",
+        disable: list = [],
```

Comment:
Having mutable default arguments can lead to weird and unexpected behavior; could you default these to `None` and do a simple `self.disable = disable or []` in the body of the method?  Same for `component_cfg`

Reply:
Good catch! Fixed in latest commits